docker/dev/centos/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM centos:centos6
+
+MAINTAINER FIWARE Orion Context Broker Team. Telefónica I+D
+
+WORKDIR /root
+
+# Install dependencies for building
+RUN \
+	yum -y install git make cmake gcc-c++ scons libmicrohttpd-devel boost-devel libcurl-devel tar wget bzip2
```

Comment:
one best practice suggested by Docker is to include one per line when several dependencies are installed. and to order them alphabetically as I did in the Ubuntu docker